cryptol-remote-api/src/CryptolServer/Data/Expression.hs

```diff
@@ -138,10 +137,11 @@ instance JSON.FromJSON ExpressionTag where
       "record"         -> pure TagRecord
       "sequence"       -> pure TagSequence
       "tuple"          -> pure TagTuple
+      "let"            -> pure TagLet
       "call"           -> pure TagApp
       "instantiate"    -> pure TagTypeApp
       "integer modulo" -> pure TagIntMod
-      "unique name"    -> pure TagUniqueName
+      "variable"       -> pure TagVariable
```

Comment:
Make sure to document these changes in the `Described Expression` instance.

Reply:
Thanks - I actually completely forget about adding docs for this new (now modified) constructor previously. Now fixed! =)